doc/api/tls.md

```diff
@@ -789,8 +790,6 @@ added: v0.11.3
 * `port` {number}
 * `host` {string}
 * `options` {Object}
-  * `host` {string} Host the client should connect to.
```

Comment:
This is still supported. We can pass `host` and `port` either directly or by setting them in `options`. In this case, both `host` and `port` are optional.